src/OmniSharp.Host/CommandLineApplication.cs

```diff
@@ -117,5 +123,19 @@ private void DebugAttach()
                 }
             }
         }
+
+        public string Locale => _locale.GetValueOrDefault(string.Empty);
+
+        private void SetLocale()
+        {
+            if (!string.IsNullOrEmpty(Locale))
+            {
+                try
+                {
+                    CultureInfo.CurrentUICulture = CultureInfo.GetCultureInfo(Locale);
+                }
+                catch (CultureNotFoundException) { }
```

Comment:
Can we log a warning or something?

Reply:
This is before we have a logger. Could use `Console.WriteLine`, but unsure how that might affect the STDIO host.

Reply:
Could we use `Console.Error.WriteLine`? Presumably the stdio hosts aren't looking for response output from that?

Reply:
Good call.

Reply:
Updated